taraxa/state/state_db/block_reader.go

```diff
@@ -55,27 +55,27 @@ func (self ExtendedReader) ForEachStorage(addr *common.Address, f func(*common.H
 	})
 }
 
-func (self ExtendedReader) ForEachAccountNodeHashByRoot(storage_root *common.Hash, f func(*common.Hash)) {
+func (self ExtendedReader) ForEachAccountNodeHashByRoot(storage_root *common.Hash, f func(*common.Hash, []byte)) {
 	if storage_root == nil {
 		return
 	}
 	no_addr := common.Address{}
 	trie.Reader{AccountTrieSchema{}}.ForEachNodeHash(
 		AccountTrieInputAdapter{&no_addr, self},
 		storage_root,
-		func(hash *common.Hash) {
-			f(hash)
+		func(hash *common.Hash, b []byte) {
```

Comment:
Why its not passed as just `f`, but wrapped in `func` again? 
Same thing for the [line 78](https://github.com/Taraxa-project/taraxa-evm/pull/165/files#diff-eb8b89da740943ea9142ba1c6de989f73ecca1b8c52798b5cd21923fb2daa054R78)

Reply:
Fixed